src/GeositeFramework/js/SubRegion.js

```diff
@@ -146,7 +146,15 @@
         _.each(subregions.areas, function(subregion) {
             var geom = new esri.geometry.Polygon(subregion.shape);
             var symbol = new esri.symbol.SimpleFillSymbol();
+            var outline = new esri.symbol.SimpleLineSymbol(
+                    esri.symbol.SimpleLineSymbol.STYLE_SOLID,
+                    new dojo.Color(subregion.outlineColor || subregions.outlineColor), 2);
```

Comment:
If both `subregion` and `subregions` don't have `outlineColor` defined, does this get an error or some default inside of dojo?